GVRf/Framework/framework/src/main/java/org/gearvrf/GVRActivityBase.java

```diff
@@ -123,6 +121,7 @@ public void run() {
     abstract GVRActivityNative makeActivityNative();
     abstract GVRViewManager makeViewManager(GVRXMLParser xmlParser);
     abstract GVRMonoscopicViewManager makeMonoscopicViewManager(GVRXMLParser xmlParser);
+    abstract GoogleVRViewManager makeGoogleVRViewManager(GVRXMLParser xmlParser);
```

Comment:
It is an artifact of the incomplete separation. This file is present in all backends and I will add this new API everywhere if that is what you choose do.

I am now in the process of gradually removing the dependencies from the framework to the backends. So GVRConfigurationManager and its location will change in the future.